src/app/Game.ts

```diff
@@ -50,6 +76,10 @@ export namespace Game {
         return currentInventory;
     }
 
+    export function getCombatState(): CombatState {
```

Comment:
this global state it going to make it difficult to do serialization. all the mutable parts of game state should be in an object so that we can give that object an Augustus schema

Reply:
There's now an overarching `GameState` tracker to keep all the little state trackers corralled.

Reply:
it's still better to just use ES6 modules instead of the `namespace` keyword, but this is acceptable.